website/versioned_docs/version-0.11/plugins/joblib.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
This should be in the top level docs, not the version-0.11 snapshot.
you can view the docs there through url like:
http://localhost:3000/docs/next/intro

Can you update the README for website to mention this? versioning is new in the website and this one keeps confusing people.

Reply:
Added a note at the bottom of README